### PR TITLE
Splunk salt module

### PR DESCRIPTION
**Pull request by yashsarma**

### CODE REVIEW

File: salt/modules/splunk.py [hunk 1]
```diff
@@ -0,0 +1,309 @@
+# -*- coding: utf-8 -*-
+'''
+Module for interop with the Splunk API
+
+.. versionadded:: 2015.2
```
Comment (jfindlay) [inline salt/modules/splunk.py]: This needs to be `Boron`.

File: salt/states/splunk.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+'''
+Splunk User State Module
+
+.. versionadded:: 2015.2
```
Comment (jfindlay) [inline salt/states/splunk.py]: Same here.

**Comment by jfindlay** [comment]
@yashlyft, awesome, thanks.

**Comment by yashlyft** [comment]
@jfindlay :+1:

**Pull request merged**